src/error.rs

```diff
@@ -255,6 +290,28 @@ impl<Src, Dst: ?Sized> AlignmentError<Src, Dst> {
     }
 }
 
+impl<Src, Dst: ?Sized + Unaligned> AlignmentError<Src, Dst> {
+    fn into_infallible(self) -> Infallible {
```

Comment:
Slap an `#[inline(always)]` on this — we want the effects of `unreachable_unchecked` to propagate up the call chain.

Reply:
Done